### DISCUSSION TITLE
Aggiornamento firmware Conbee2

### DISCUSSION THREAD

**Discussion by gforment** [indomus/forum#280] [HUB: Home Assistant]
Ciao,
perdonate se sono sempre a chiedere cose magari anche banali, prometto che alla prima occasione restituirò il favore!
Nell'attività di re-installazione che sto portando avanti (HA su RPI4 con docker compose) sono arrivato a Zigbee che vorrei gestire con ZHA (ho esperienza precedente con Z2M su Docker).
Passo 1 è l'**aggiornamento del Conbee2** che ho da qualche mese a "Tortuga" (deCONZ_ConBeeII_0x266b0700.bin.GCF), tenendo presente che in casa non ho un PC Windows da usare per un flash standalone. La strada che prendo è quindi quella di **istanziare un container usando "marthoc/deconz"** da tenere solo per effettuare gli aggiornamenti perchè il comando GCFFlasher_internal indicato nella guida InDomus non mi pare disponibile senza installare deCONZ (https://indomus.it/formazione/utilita-come-aggiornare-deconz-e-il-firmware-conbee-raspbee/).
Creato il container, qui sorge il mio dubbio:
- perchè nella schermata della web interface, sotto Settings > Gateway sembra tutto aggiornato? deconz ok, ma il firmware è indicato 26660700 (versione di ottobre 2020)
- se seguo la guida specifica di marthoc su GitHub https://github.com/marthoc/docker-deconz#1-check-your-deconz-container-logs-for-the-update-firmware-file-name, nei log non viene visto il nuovo aggiornamento
```
22:15:08:197 GW update firmware found: /usr/share/deCONZ/firmware/deCONZ_ConBeeII_0x26660700.bin.GCF
22:15:08:197 GW firmware version: 0x26660700
22:15:08:197 GW firmware version is up to date: 0x26660700
``` 
Avete indicazioni? Procedo con un upgrade "manuale" con firmware-update.sh presente nel container puntando al nuovo firmware ossia http://deconz.dresden-elektronik.de/deconz-firmware/deCONZ_ConBeeII_0x266b0700.bin.GCF?
`docker run -it --rm --entrypoint "/firmware-update.sh" --privileged --cap-add=ALL -v /dev:/dev -v /lib/modules:/lib/modules -v /sys:/sys marthoc/deconz`
Grazie!

**Comment by indomus** [comment]
Perché purtroppo deCONZ è diventato poco affidabile sulla gestione fw, non si sa bene perché.
Sì, procedi serenamente "a mano", indicando il firmware nuovo.

**Reply by gforment** [reply]
Rapido e preciso, grazie! Magari può essere utile indicarlo nell'articolo o integrare questo aspetto per i meno smaliziati
